src/docfx/config/Config.cs

```diff
@@ -90,12 +88,16 @@ private static Config LoadCore(string configPath, CommandLineOptions options)
             try
             {
                 var configObject = Expand(YamlUtility.Deserialize<JObject>(File.ReadAllText(configPath)) ?? new JObject());
+                configObject.Merge(options.ToJObject(), JsonUtility.DefaultMergeSettings);
```

Comment:
yeah, but that one produces a new object rather than merging into an existing object.

---
In reply to: [187517363](https://github.com/dotnet/docfx/pull/2758#discussion_r187517363) [](ancestors = 187517363)